test/support/organisation_helper.rb

```diff
@@ -701,12 +701,8 @@ def setup_and_visit_page(content)
     visit content["base_path"]
   end
 
-  def i_see_the_current_page_breadcrumb
-    assert page.has_css?(".gem-c-breadcrumbs", text: @title)
-  end
-
-  def and_the_courts_breadcrumb
-    assert page.has_css?(".gem-c-breadcrumbs", text: "Courts and Tribunals")
+  def i_see_the_courts_breadcrumb
+    assert page.has_css?(".gem-c-breadcrumbs")
```

Comment:
This needs to actually test what the test name suggests, i.e: look for the courts breadcrumb text.

Reply:
not sure what to do with this as the breadcrumb would only display "Home > Organisation" 

what do suggest?

Reply:
Replied on your previous comment.